sql/core/src/test/resources/sql-tests/inputs/pipe-operators.sql

```diff
@@ -71,6 +71,456 @@ create temporary view windowTestData as select * from values
   (3, 1L, 1.0D, date("2017-08-01"), timestamp_seconds(1501545600), null)
   AS testData(val, val_long, val_double, val_date, val_timestamp, cate);
 
+create table store_sales(
```

Comment:
Shall we move the TPCDS table creation/drop to the `SQLQueryTestSuite`, so that we can review the outputs easier.
We can have something like `def requireTPCDSCases: Seq[String]` to define which golden file needs the TPCDS table definitions.

Reply:
Sure, this is done. I found an existing trait `TPCDSSchema` that can help with this.